### DISCUSSION TITLE
support for Windows on ARM64 (native).

### DISCUSSION THREAD

**Discussion by zedd-0** [rustdesk/rustdesk#5532] [Q&A] [answered]
it would be nice, if there were a native arm64 version for windows.
(client & server)

thx.

**Comment by xlionjuan** [comment]
Need for client too

Server already has arm64 version

**Comment by cheezmil** [comment]
+1

**Comment by ounrequitedo** [comment]
+1

**Comment by talynone** [comment]
Another vote for a ARM64 Windows native build. It seems to work fine under the PRISM emulation layer, but there are quirks here and there I'm wondering if they may be related to it no being native (though they may not be). One quirk I encountered was after a while no app would allow audio to be produced until I exited the RustDesk client, though this may have been just a fluke as I haven't been able to re-produce that issue.

**Comment by chriswhaas** [comment]
another vote for Windows Arm64 from me. Thank you

**Comment by xlionjuan** [comment]
It should be possible if GitHub-hosted ARM runner become general available on free plan.

Which will be available on the end of the year.

**Reply by talynone** [reply]
Looks like "ARM hosted runners for OSS/free plans [Public Preview] #970" (https://github.com/github/roadmap/issues/970) has been put on the roadmap for Q1 2025 : https://github.com/orgs/github/projects/4247

**Reply by xlionjuan** [reply]
Initially planned at the end of the 2024, but postponed to 2025 Q1.

**Reply by talynone** [reply]
Linux arm64 hosted runners now available for free in public repositories (Public Preview)

https://github.blog/changelog/2025-01-16-linux-arm64-hosted-runners-now-available-for-free-in-public-repositories-public-preview/

**Reply by xlionjuan** [reply]
This is Linux, not Windows.

**Comment by pgs666** [comment]
+1

**Comment by peter812** [comment]
+1

**Comment by ParadiseCity1986** [comment]
+1

**Comment by DoM1niC** [comment]
+1

**Comment by hayd1n** [comment]
+1

**Comment by PuNkYsHuNgRy** [comment]
+1

**Comment by raphlafalaf** [comment]
+1 ! I think it was supposed to be developed for Q1 25, are there any updates?

**Comment by fanjofo** [comment]
+1

**Comment by talynone** [comment]
@xlionjuan Public Windows ARM GitHub runners now available:

https://blogs.windows.com/windowsdeveloper/2025/04/14/github-actions-now-supports-windows-on-arm-runners-for-all-public-repos/

**Comment by MeDavoo** [comment]
+1

**Comment by SkipToTheEndpoint** [comment]
+1

**Comment by AppliedEllipsis** [comment]
+1

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/pull/15139

**Reply by lithStudy** [reply]
support for Windows on ARM is now in the stable release for dart and included in flutter. It should now be able to build with dart 3.9.3:
https://dart.dev/get-dart/archive#stable-channel

**Comment by Cd084** [comment]
+1

**Comment by ifalatiksetlog** [comment]
support for Windows on ARM is now in the stable release for dart and included in flutter. It should now be able to build with dart 3.9.3:
https://dart.dev/get-dart/archive#stable-channel

**Comment by SPaGue67** [comment]
+1

**Comment by jkelleyrtp** [comment]


**Reply by stevej-algoteer** [reply]
Please do! You'd be our hero. :)

**Comment by punteroanull** [comment]
Is there any update about this? Thank you, guys.

**Comment by AverageThe** [comment]
+1

**Comment by ushabuntu** [comment]
+1

**Comment by denisemenov** [comment]
+1

**Comment by moose999** [comment]
+1

**Comment by lithStudy** [comment]
+1,please

**Comment by Phoenixy** [comment]
+1

**Comment by dennisameling** [comment]
Here's a WIP pull request to add Windows arm64 support: https://github.com/rustdesk/rustdesk/pull/15139

I managed to get a basic working build locally which is available here: https://github.com/dennisameling/rustdesk/releases/tag/win-arm64

**Comment by stevej-algoteer** [comment]
I can't wait to try it out in an hour. Thanks!

**Comment by dennisameling** [comment]
With https://github.com/rustdesk/rustdesk/pull/15139 being merged, Windows arm64 is now included in the nightly builds (`.exe` and `.msi`) 🚀 

https://github.com/rustdesk/rustdesk/releases/tag/nightly

**Comment by dennisameling** [comment]
Version 1.4.8 has been released which includes Windows arm64 support: https://github.com/rustdesk/rustdesk/releases/tag/1.4.8

**Reply by Mik-c-OS** [reply]
thanks a lot.

**Comment by zedd-0** [comment]
goal achieved.
thank you all so much.
/closed
